### ISSUE TITLE
Traceback : TypeError: can only concatenate str 

### ISSUE THREAD

**Issue by octavsly** [WKnak/fail2ban-block-ip-range#6]
Maybe the following error report can be used...
```
Traceback (most recent call last):
  File "/usr/local/bin/fail2ban-block-ip-range.py", line 116, in <module>
    print("jail " + jail + " unsuccessful try to ban aggregated IP network: " + ip + " (result: " + result + ")")
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
TypeError: can only concatenate str (not "CompletedProcess") to str
```

**Comment by pbiering** [comment]
reproduced, will check for the issue

**Comment by pbiering** [comment]
issue was introduced by https://github.com/WKnak/fail2ban-block-ip-range/pull/3/files

**Comment by WKnak** [comment]
Due to some failed PR merges from my side, it is now fixed my PR #11 from @pbiering. Thanks!
